engines/stark/ui/window.cpp

```diff
@@ -38,6 +38,12 @@ Window::Window(Gfx::Driver *gfx, Cursor *cursor) :
 Window::~Window() {
 }
 
+Common::Rect Window::getScaledPosition() const {
+  return Common::Rect(_gfx->scaleWidthOriginalToCurrent(Gfx::Driver::kGameViewportWidth),
```

Comment:
Please indent your code with tabs (throughout). Our code formatting conventions are there: http://wiki.scummvm.org/index.php/Code_Formatting_Conventions

Reply:
`Window` objects have their own position in the `_position` variable. This method needs to return the window's position not the game viewport position. And `ActionMenu` needs to call `getScaledPosition` for the GameWindow instance.